cypress/integration/test_table.js

```diff
@@ -173,16 +193,57 @@ describe("Tests Table", function() {
     });
 
     it("Input value is included in the taskTests GQL request body under variables.testName ", () => {
-      const xhrTestNamePath = "requestBody.variables.testName";
-      waitForGQL("@gqlQuery", "taskTests", {
-        [xhrTestNamePath]: testNameInputValue
-      });
-      cy.get("@gqlQuery")
-        .its("requestBody.operationName")
-        .should("equal", "taskTests");
-      cy.get("@gqlQuery")
-        .its(xhrTestNamePath)
-        .should("equal", testNameInputValue);
+      assertQueryVariables("STATUS", "ASC", [], testNameInputValue, 0);
+    });
+  });
+
+  describe("Scrolling", () => {
+    beforeEach(() => {
+      cy.visit(TESTS_ROUTE);
+      assertQueryVariables();
+    });
+
+    it("Fetches and appends additional tests to table as the user scrolls", () => {
+      cy.get(".ant-table-row")
+        .invoke("toArray")
+        .then($initialTasks => {
+          // need to overscroll to trigger fetch
+          cy.get(".ant-table-body").scrollTo(0, "101%", { duration: 500 });
+          assertQueryVariables("STATUS", "ASC", [], "", 1);
+          cy.get(".ant-table-row").should(
+            "have.length.greaterThan",
+            $initialTasks.length
+          );
+        });
```

Comment:
in future, i think it would be beneficial for pages/features of the app that are more or less identical to be tested in exactly the same way. for example, it's interesting that tests for this feature were already written via the task table scrolling tests. but you chose to rewrite the tests in a slightly different way. if you think this way is better then should this approach be generalized and applied to the task table? it seems inefficient to be rewriting tests for features that exactly the same give or take a url parameter.

Reply:
good point